index.js

```diff
@@ -32,20 +54,22 @@ async function getIssue() {
 
 async function doStuff() {
     const issue = await getIssue();
-    const sections = issue.body.split('###');
+    const issueBody = issue.body || '';
+    const sections = issueBody.split('###');
     const content = {};
+
     for(const section of sections) {
         const [ title, ...body ] = section.split('\n');
-        content[title.trim()] = body.join("\n").trim();
+        const key = getPropertyFromHeadingMapping(title.trim());
+        content[key] = body.join("\n").trim();
     }
-    if(content.hasOwnProperty(core.getInput('scheduledTitle'))) {
-        const rawDate = content[core.getInput('scheduledTitle')];
-        const scheduledDate = new ScheduledDate(rawDate, core.getInput('timezone'), core.getInput('dateFormat'));
+    if(content.hasOwnProperty('scheduledDate')) {
+        const scheduledDate = new ScheduledDate(content.scheduledDate, inputs.timezone, inputs.dateFormat);
         content.date = {
             timestamp: scheduledDate.getTime(),
-            valid: scheduledDate.valid,
```

Comment:
why not output this anymore?

Reply:
My thinking here was that `cardContent` should just be the content without validation. This makes part of `ScheduledDate` obsolete, and I didn't remove that part. Happy to re-add it here or remove the valid field entirely, up to you :)

Reply:
Since this is called "parse" it should absolutely pass on errors detected during parsing. It might not handle them (unless told to), but it should still output them.